src/container/cli.c

```diff
@@ -840,11 +839,10 @@ cont_close_complete(tse_task_t *task, void *data)
 		" master\n", DP_CONT(pool->dp_pool, cont->dc_uuid),
 		arg->hdl.cookie, DP_UUID(cont->dc_cont_hdl));
 
+	daos_csummer_destroy(&cont->dc_csummer);
```

Comment:
Moving this call is unnecessary, for there's a d_list_del_init call that also uses the cont reference after the dc_cont_put(cont) two lines below.

My reading is:

    dc_cont_open +1 for cont and transfers ref to task priv
    cont_open_complete adds cont to pool without ref
    cont_open_complete +1 for handle hash table
    cont_open_complete +1 for handle
    cont_open_complete -1 for cont

    dc_cont_close +1 for cont
    cont_close_complete -1 for handle hash table
    cont_close_complete -1 for handle
    cont_close_complete removes cont from pool without ref
    cont_close_complete -1 for cont